OpenOversight/tests/test_commands.py

```diff
@@ -1,14 +1,31 @@
 import csv
+import datetime
+import operator
+import os
 import traceback
 import uuid
 
+from click.testing import CliRunner
 from sqlalchemy.orm.exc import MultipleResultsFound
 
 import pandas as pd
 import pytest
-from click.testing import CliRunner
-from OpenOversight.app.commands import add_department, add_job_title, bulk_add_officers
-from OpenOversight.app.models import Assignment, Department, Job, Officer, Salary
+from OpenOversight.app.commands import (
```

Comment:
Did you make these formatting changes due to personal preference?  Or did flake8 request them?  I'm not sure if I'm a fan of putting every `command` and `model` import on a separate line.

Reply:
I was using the auto-formatter "Black" on that file, I am assuming it decided that the line will be too long with all those imports. I don't have an opinion on this, but I do hope to adapt Black as our formatting standard at some point: https://github.com/lucyparsons/OpenOversight/issues/746